website/client/src/en/guide/configuration.md

```
@@ -58,10 +59,11 @@ Location:
 ## Ignore Patterns
 
 Priority:
-1. CLI options (`--ignore`)
-2. `.repomixignore`
-3. `.gitignore` and `.git/info/exclude`
-4. Default patterns
+1. Force include patterns (`--force-include`)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)